setup.py

```diff
@@ -37,8 +37,12 @@
 with open(os.path.join(this_directory, 'README.md'), encoding='utf-8') as f:
     LONG_DESCRIPTION = f.read()
 
-
 if 'HPY_DEBUG' in os.environ:
+    print("WARNING: environment variable HPY_DEBUG used in setup.py was "
+          "renamed to HPY_DEBUG_BUILD. HPY_DEBUG is now used to activate "
+          "the HPy debug context when running HPy extensions in universal mode.")
+
```

Comment:
I think this is not needed. `HPY_DEBUG` was just an internal only thing used by our Makefile, I don't think it's used by anyone else.
And, btw, you should modify the `Makefile` so that `make debug` still works :)

Reply:
done